### ISSUE TITLE
Remove the "-KU" options from shebangs

### ISSUE THREAD

**Issue by voxik** [zzak/sdoc#126]
Shebangs are specifying ```-KU``` option:

~~~
$ find . -executable -type f
./bin/sdoc-merge
./bin/sdoc

$ find . -executable -type f -exec head -1 {} \;
#!/usr/bin/env ruby -KU
#!/usr/bin/env ruby -KU
~~~

If I am not mistaken, this was Ruby 1.8.x era option, but SDoc does not support Ruby 1.8 since https://github.com/zzak/sdoc/commit/c28efc8d4ebe0a50ed1072f3e4041657b149b634, so this options should be dropped.

**Comment by voxik** [comment]
BTW I get this failure on my system:

~~~
$ LANG=C.UTF-8 bin/sdoc
/usr/bin/env: ‘ruby -KU’: No such file or directory

$ cat /etc/system-release
Fedora release 29 (Rawhide)
~~~

**Comment by robin850** [comment]
Resolved by #129 ; thanks for reporting @voxik !
